### DISCUSSION TITLE
PLY to OBJ

### DISCUSSION THREAD

**Discussion by chekhroma** [cnr-isti-vclab/PyMeshLab#30] [Q&A]
Hello. I am trying to convert the PLY with texture to OBJ format, but I am having some issues. First, the obj file loses its UV and later it is not possible to use the texture on it. Secondly, when i am saving obj, no mtl file is created. When i use desktop version of MeshLab everything nice. My sample code: 
```
import pymeshlab as ml
ms = ml.MeshSet()

ms.load_new_mesh('model.ply')
ms.apply_filter('set_texture', textname = 'model.jpg', textdim=1024)
ms.save_current_mesh('2.obj', save_vertex_color=True, save_vertex_coord=True, save_vertex_normal=True, save_face_color=True, save_wedge_texcoord=True, save_wedge_normal=True, save_polygonal=True)

```

**Comment by alemuntoni** [comment]
Hi, did you tried to give the absolute path to the `textname` parameter? 
There could be some issue on that..
maybe setting verbosity to True could help to understand the issue:
```
ms = ml.MeshSet(True)
```
